test/dbus/util-broker.c

```diff
@@ -318,7 +321,8 @@ void util_fork_daemon(sd_event *event, int pipe_fd, pid_t *pidp) {
                 c_assert(r >= 0);
 
                 /* exec dbus-daemon */
-                bin = getenv("DBUS_BROKER_TEST_DAEMON") ?: "/usr/bin/dbus-daemon";
+                bin = getenv("TEST_DBUS_DAEMON") ?: "/usr/bin/dbus-daemon";
```

Comment:
If we drop `TEST_DBUS_BROKER`, I'd be fine with making this a boolean env-var and using `BINDIR "/dbus-daemon"`.